packages/@angular/cli/bin/ng

```diff
@@ -86,6 +86,30 @@ if (require('../package.json')['name'] == 'angular-cli'
   `));
 }
 
+const packageJsonProjectPath = path.join(process.cwd(), 'package.json')
```

Comment:
We allow people to run anywhere inside a project. You should use the `__dirname` instead (and potentially `findup`).